start_cerberus.py

```diff
@@ -206,6 +220,12 @@ def main(cfg):
                         logging.info("Failed containers in %s: %s", pod, containers)
                 logging.info("")
 
+            # Logging the failed checking of routes
+            if failed_routes:
+                logging.info("Iteration %s: Failed route monitoring" % iteration)
+                for route in failed_routes:
+                    logging.info("Route url: %s" % route)
+
```

Comment:
Please add logging.info("") after logging all the failed routes. This ensures the separation of different component failures that are logged in the report, thereby improving the readability.